pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
@The-Compiler 
Well:

> % git shortlog -sn
   122  Bruno Oliveira
     4  Rafael Bertoldi (fogo)
     3  Jurko Gospodnetić
     2  Florian Bruhin
     2  Pavel Savchenko
     2  Tiago Nobrega
     2  jhermann

;)

But it's under the pytest-dev umbrella of course, so you are right in that regard.

new-style classes however are only relevant for py2, and I was disregarding this by ignoring old-style things.. ;)